sdk/communication/azure-communication-identity/pom.xml

```diff
@@ -171,5 +172,47 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>disable-coverage-if-phonenumbers</id>
+      <activation>
```

Comment:
instead of defining the profile multiple times in different packages, it might be better to have a ```profiles.xml``` to contain all this settings and you can just reference them here using ids